target/linux/ar71xx/base-files/lib/ar71xx.sh

```diff
@@ -963,6 +963,9 @@ ar71xx_board_detect() {
 	*TEW-823DRU)
 		name="tew-823dru"
 		;;
+	*"TL-WR942N v1")
```

Comment:
> Lines from 965 to 1136 are not in alphabetical order. Can I sort all boards in this range?

I know about broken order, I plan to fix it later, for now please just put your board somewhere under TL-WR941...